### PR TITLE
55 document all ros packages pt. 1

### LINKED ISSUE #55: Document all ROS packages

### PR DESCRIPTION
**Pull request by bennowotny**
Closes #55

### CODE REVIEW

File: src/wr_control_drive_science/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# wr_control_drive_science
+
+@defgroup wr_control_drive_science wr_control_drive_science
+
+## Hardware Description (move me to wr_control_drive_science)
```
Comment (srao32) [inline src/wr_control_drive_science/README.md:5]:
```suggestion
## Hardware Description
```

**Comment by srao32** [comment]
For some reason, the collaboration diagrams don't show up for the modules under wrover_control_drive_science.

**Review by srao32 — CHANGES_REQUESTED** [review]

File: src/wroboarm_23/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# wroboarm_23
+
+@defgroup wroboarm_23 wroboarm_23
+
+This package contains the model of the robotic arm on the front of the WRover, as well as the required information for MoveIt to perform IK on the arm.  
+
+## Editing
+
+### The package
+
```
[the remark this answers is not in the document]
Comment (bennowotny) [reply src/wroboarm_23/README.md:10]: Addressed in d9035cd.

File: src/wroboarm_23/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+The model relies on an exported URDF definition of the arm.  This is generated via a plugin to Solidworks, which is provided on CoE lab machines.  
+
+For the export, to avoid load on the IK system while maintaining simulation integrity, use actual parts of the arm (plates, motors, structural components), but exclude small components that do not significantly affect the work area (like nuts, bolts).
+
+The `config` folder has configuration parameters not known to the model itself, like joint limits.
+
+## Launching
+
+The `demo.launch` file is autogenerated, and `demo_test.launch` is a slightly modified version of that file made to launch correctly across the WRover system.
```
[the remark this answers is not in the document]
Comment (bennowotny) [reply src/wroboarm_23/README.md]: Addressed in d9035cd.

**Review by srao32 — APPROVED** [review]

**Comment by bennowotny** [comment]
This PR will not merge.  When the story is really complete, we will re-review and merge then.

**Pull request closed without merging**